specifyweb/frontend/js_src/lib/components/DataModel/types.ts

```diff
@@ -5319,6 +5319,7 @@ export type SpQueryField = {
     readonly formatName: string | null;
     readonly isDisplay: boolean;
     readonly isNot: boolean;
+    readonly isStrict: boolean;
```

Comment:
Looks good! If not done already, can you pull production and regenerate frontend types to make sure we don't lose any types that exist on prod right now?